packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -30,7 +30,7 @@ class WalletsController {
   }
 
   public static index = (): ChannelResponse<WalletData[]> => {
```

Comment:
It seems that this method has been missed in the refactor.

Reply:
yes